src/components/Form/FormTextarea.react.js

```diff
@@ -32,7 +34,9 @@ function FormTextarea({
       value={value}
       disabled={disabled}
       rows={rows}
-    />
+    >
+      {children}
```

Comment:
Textareas in React should only use a value prop rather than children, perhaps do `value={value || children} so that our own FormTextarea can still handle children?

Reply:
I feel like the option you are proposing would work well, will implement shortly.